CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.1.0]
+### Added
+- Add `--autoCategorize` flag to `update` command ([#212](https://github.com/MetaMask/auto-changelog/pull/212))
+  - When populating the Unreleased section, the tool will look for [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) prefixes in commit message subjects and place matching commits in corresponding categories automatically.
```

Comment:
I already approved however I am not sure subject is the right keyword here. Can we drop it the subject is usually what goes after the prefix ? Or I am wrong ?

Reply:
"commit message subject" sometimes means the first line of a commit message, which would include the prefix we're looking for here.

Reply:
Thanks for the explanation @Gudahtt